client/src/main.ts

```diff
@@ -173,8 +173,17 @@ async function _startServer(extensionUri: vscode.Uri, supportedLanguages: Suppor
 		...Object.keys(vscode.workspace.getConfiguration('files', null).get('exclude') ?? {})
 	].join(',')}}`;
 
-	const size = Math.max(0, vscode.workspace.getConfiguration('anycode').get<number>('symbolIndexSize', 100));
-	const init = Promise.resolve(vscode.workspace.findFiles(langPattern, exclude, size + 1).then(async uris => {
+	let size = Math.max(0, vscode.workspace.getConfiguration('anycode').get<number>('symbolIndexSize', 100));
+	// do not set a maxResults limit if RemoteHub has the full workspace contents
+	const remoteHub = vscode.extensions.getExtension('GitHub.remoteHub') ?? vscode.extensions.getExtension('GitHub.remoteHub-insiders');
+	const remoteHubApi = await remoteHub?.activate();
+	let hasFullWorkspaceContents = false;
+	if (remoteHubApi && remoteHubApi.hasWorkspaceContents && vscode.workspace.workspaceFolders) {
+		for (const folder of vscode.workspace.workspaceFolders) {
+			hasFullWorkspaceContents = hasFullWorkspaceContents && (await remoteHubApi.hasWorkspaceContents(folder.uri));
```

Comment:
This will stay `false`, right? It's like `X = false && Y`?

Reply:
Thanks! Fixed this in my test extension and forgot to update the actual PR. Sorry about that.